samples/angular-todo/src/app/shared/idps.enums.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * @forgerock/javascript-sdk
+ *
+ * idps.enums.ts
+ *
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+/**
+ * Types of identity providers
+ */
+enum IdpNames {
```

Comment:
I have a preference that we use Unions instead of enums, as a best practice. Can we convert this to a union type?